### DISCUSSION TITLE
An error message occurred when I added a resource to a new dataset.

### DISCUSSION THREAD

**Discussion by hongjie123** [ckan/ckan#7080] [General]
Try adding a JSON file for the new dataset, as shown below:
![image](https://user-images.githubusercontent.com/54384385/189586373-78e10023-3b6b-455e-954d-61f1842b300b.png)

An error message occurred when I added a resource to a new dataset. The diagram below:
![image](https://user-images.githubusercontent.com/54384385/189585624-81af1b37-414c-4a55-8535-1e70b62e50aa.png)
This error only occurs when adding JSON files. TXT, RDF, and XML files can be added successfully.
Trying to add a JSON file with a different new dataset keeps getting this error.

**Comment by wardi** [comment]
There's not enough information to work on this ticket so I'm converting it to a support discussion. Uploading a JSON file to vanilla CKAN does work properly.

Try disabling some of the plugins you have installed that could be causing the issue, and provide the server logs from when this error occurs. A member of the community might be able to help you with this.

**Reply by hongjie123** [reply]
The log files are as follows:
![image](https://user-images.githubusercontent.com/54384385/190040204-8378c773-30b6-46a2-a217-d31d4ea9ac69.png)

![image](https://user-images.githubusercontent.com/54384385/190040327-383097f7-cde8-41d9-881d-78d502f67c26.png)


![image](https://user-images.githubusercontent.com/54384385/190040387-a18698a9-7e3e-454a-8514-698817ec2562.png)
